features/export.feature

```diff
@@ -74,6 +74,58 @@ Feature: Export content.
       10
       """
 
+  Scenario: Export a comma-separated list of post types
+    Given a WP install
+
+    When I run `wp plugin install wordpress-importer --activate`
+    Then STDERR should not contain:
```

Comment:
You should always test what you're expecting — `Then STDOUT should contain.. Success:`